lib/web_ui/lib/src/engine/semantics/scrollable.dart

```diff
@@ -99,12 +100,12 @@ class Scrollable extends RoleManager {
 
   @override
   void update() {
-      // We neutralize the scroll position after all children have been
-      // updated. Otherwise the browser does not yet have the sizes of the
-      // child nodes and resets the scrollTop value back to zero.
-      semanticsObject.owner.addOneTimePostUpdateCallback(() {
-        _neutralizeDomScrollPosition();
-      });
+    // We neutralize the scroll position after all children have been
+    // updated. Otherwise the browser does not yet have the sizes of the
+    // child nodes and resets the scrollTop value back to zero.
```

Comment:
This comment is no longer accurate and can be removed. After this change we can set `scrollTop` to a non-zero value even on empty containers. We should find out why tests are failing when `addOneTimePostUpdateCallback` is removed, and if there's a valid reason then document it, otherwise update tests. This can be done later. No need to block this PR.

Reply:
Agreed!